lib/src/input/figma/entities/layers/figma_node.dart

```diff
@@ -56,6 +61,37 @@ class FigmaNode {
     this.layoutGrow,
   });
 
+  @JsonKey(ignore: true)
+  FigmaStyleProperty figmaStyleProperty;
+
+  static Map figmaStylePropertyToJson(FigmaStyleProperty property) {
+    var map = {};
+    // TODO: implement
+    return map;
+  }
+
+  static FigmaStyleProperty figmaStylePropertyFromJson(Map json) {
```

Comment:
Should this be called using the JSON Serializable as well? I see that FigmaStyleProperty has a `fromJson` method

Reply:
I forgot but that's true, I just fixed it